index.js

```diff
@@ -397,6 +402,31 @@ p.setRoomData = function( roomID, data ) {
 };
 
 /**
+ * Set room as public
+ * 
+ * @param  {String} roomID id for the room you'd like to make public
+ * @return {Promise} This promise will succeed when the room as been set as public
+ */
+p.makeRoomPublic = function( roomID ) {
```

Comment:
Again I wonder if it would be better to have public rooms and then users join private rooms. It would decrease the possibility of race conditions.